spec/controllers/sessions_controller_spec.rb

```diff
@@ -77,6 +172,24 @@
       it { should redirect_to_url_after_destroy }
     end
 
+    context "with a domain cookie" do
+      before do
+        with_custom_config(:cookie_domain, '.example.com'){
+          user = create(:user)
+          user.update_attribute :remember_token, "old-token"
+          post :create, session: { email: user.email, password: user.password }
+          request.cookies["remember_token"] = { value: "old-token", domain: '.example.com' }
+          delete :destroy
+        }
+      end
+
+      it "should delete the domain cookie" do
+        expect(response.cookies['remember_token']).to be_nil
+        expect(response.headers['Set-Cookie']).to match('domain=.example.com')
+        expect(response.headers['Set-Cookie']).to match(/Jan.1970/)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.